config/prod/usage-statistics-S3.yaml

```diff
@@ -1,6 +1,14 @@
 # For storage of files related to usage statistics reporting
-template_path: "remote/s3-bucket.yaml"
+template:
+  type: "http"
+  url: "https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/aws-infra/v0.2.7/s3-bucket.yaml"
 stack_name: "usage-statistics-S3"
+stack_tags:
+  Department: "SysBio"
+  Project: "DataEngineering"
+  OwnerEmail: 'larsson.omberg@sagebase.org'
+  CostCenter: "Other / 000001"
```

Comment:
Where's the "Other" cost center?

Reply:
it's here https://github.com/Sage-Bionetworks/aws-infra/blob/master/templates/tags/SageFinancialProgramCodes.json#L19

but this is another guess,  @larssono would you be able to suggest a cost center tag or this? options are here: https://github.com/Sage-Bionetworks/aws-infra/tree/master/templates/tags

Reply:
If @larssono is not responsive, maybe just use No Program / 00000.